visualization/src/main/scala/com/intel/analytics/bigdl/visualization/tensorboard/EventWriter.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to Intel Corporation under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * Intel Corporation licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.visualization.tensorboard
+
+import java.io.File
+import java.net.InetAddress
+import java.util.concurrent.{LinkedBlockingDeque, TimeUnit}
+
+import org.tensorflow.util.Event
+
+class EventWriter(logDir: String, flushMillis: Int = 10000) extends Runnable {
+  val eventQueue = new LinkedBlockingDeque[Event]()
```

Comment:
I think the queue should not be hold by EventWriter. It should be hold by FileWriter.